cmd/libs/go2idl/go-to-protobuf/protobuf/generator.go

```diff
@@ -206,7 +221,7 @@ func (p protobufLocator) ProtoTypeFor(t *types.Type) (*types.Type, error) {
 		return t, nil
 	}
 	// it's a message
-	if t.Kind == types.Struct {
+	if t.Kind == types.Struct || t.Kind == types.Alias {
```

Comment:
Yeah, probably.

On Tue, Jun 14, 2016 at 5:31 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In cmd/libs/go2idl/go-to-protobuf/protobuf/generator.go
> <https://github.com/kubernetes/kubernetes/pull/27258#discussion_r66937677>
> :
>
> > @@ -206,7 +221,7 @@ func (p protobufLocator) ProtoTypeFor(t *types.Type) (*types.Type, error) {
> >  		return t, nil
> >  	}
> >  	// it's a message
> > -	if t.Kind == types.Struct {
> > +	if t.Kind == types.Struct || t.Kind == types.Alias {
>
> shouldn't we do it only for "isOptionalAlias" (not for all alliases?)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27258/files/b1f617a9af71b184a197f9b70bdca6d186b36ba7#r66937677>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p5B0S78o01l43bhFIuSOqFTn9NDmks5qLnT3gaJpZM4Iz54c>
> .
>